index.html

```diff
@@ -210,6 +210,71 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
         <abbr title="application programming interface">API</abbr> properties</a> in the [[core-aam-1.2]].
       </p>
     </section>
+    <section id="mapping_contextual">
+      <h3>Exposing HTML Features With Contextual Roles</h3>
+      <p>
+        An element with a <STRONG>contextual role</STRONG> is one whose implicit WAI-ARIA role semantics are based on its ancestral <a>accessibility tree</a> relationship to 
+        another element or elements. Or, an element whose computed role is reliant on whether the element has been provided an accessible name. 
+      </p>
+      <p>
+        For instance, an element that is an <STRONG>accessibility child</STRONG> of its required <STRONG>accessibility parent</STRONG> would expose a computed role that was 
+        appropriate for its related context. Where the same element with a contextual role might be used within another markup pattern and have a different accessibility parent, 
+        and would therefore expose a computed role appropriate for that context.
```

Comment:
Word-smithing plus examples.

```suggestion
        For instance, an element that is an <strong>accessibility child</strong> of its required <strong>accessibility parent</strong> would expose a computed role for its given context, such as a list item (<code>li</code>) within an unordered list (<code>ul</code>). However, the same element might be used within a different markup pattern and could therefore expose different contextually-dependent computed roles, such as a labeled <code>section</code> element (role <code>region</code>) versus an unlabeled <code>section</code> element (role <code>generic</code>).
```

Can we think of a specific example that works for both? If so that'd be better than the sectioning content examples, which are less easy to understand and care about than a simple list.

Reply:
@cookiecrook I like these suggestions so i'm going to largely pull them in - but I'm going to be changing the use of "accessible label" back to "name".

Reply:
@cookiecrook re: other examples.

What do you think about mentioning tbody here, per the whole rowgroup conversation?  That'd be an example of a contextual role where it can be ignored until it is given a name or focusable.

Another example we could use in lieu of / in combination with the list item example is the summary of a details, where only under specific conditions is the summary an `html-summary`, and otherwise it would need to be a `role=generic`

Reply:
🤷 I could go either way. I don't think most authors give much thought to `<tbody>` etc, so I'm not sure it'd be super helpful as an example, but I also don't think it'd be harmful.